backend/db/operations.py

```
@@ -0,0 +1,55 @@
+from typing import Any
```

Ruff

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)